wandb/sdk/launch/runner/aws.py

```diff
@@ -0,0 +1,475 @@
+import configparser
+import itertools
+import logging
+import os
+import subprocess
+import time
+from typing import Any, Dict, List, Optional, Tuple
+
+if False:
+    import boto3  # type: ignore
+import wandb
+import wandb.docker as docker
+from wandb.errors import CommError, LaunchError
+from wandb.util import get_module
+
+from .abstract import AbstractRun, AbstractRunner, Status
+from .._project_spec import (
+    get_entry_point_command,
+    LaunchProject,
+)
+from ..docker import (
+    build_docker_image_if_needed,
+    construct_local_image_uri,
+    docker_image_exists,
+    docker_image_inspect,
+    generate_docker_base_image,
+    pull_docker_image,
+    validate_docker_installation,
+)
+from ..utils import PROJECT_DOCKER_ARGS, PROJECT_SYNCHRONOUS
+
+AWS_SAGEMAKER_URL = "https://{region}.console.aws.amazon.com/sagemaker/home?region={region}#/jobs/{job_name}"
```

Comment:
does this need to be a global constant? it's sorta more readable if this gets moved closer to where it gets used

Reply:
It does not. Moving it